README.md

```diff
@@ -1 +1,125 @@
-# TRAILBot
\ No newline at end of file
+TRAILBOT README 
+=====
+
+SETUP GUIDE
+=============
+Sensor Drivers:
+
+- Clone this repo and dependending repo's
+```
+cd ~/ros2_ws/src
+git clone https://github.com/TRAILab/TRAILBot.git
+git clone https://github.com/TRAILab/velodyne.git
+git clone https://github.com/TRAILab/ximea-driver.git
+cd ximea-driver
+./tools/install_sdk.sh
+cd ../..
+```
+
+
+Seperate Installations and Configurations:
+============
+Follow the husky setup readme for nav and control installations here:
+- [Link to Husky and Nav README](husky/README.md)
+Follow the build and package setup for the voice_assistant here:
+- [Link to Voice Assistant README](voice_assistant/README.md)
+Follow the build and testing setup for the human_detection here (build only if following example fully):
+- [Link to Human Detection README](human_detection/readme.md)
+Follow the download and build instructions for the FSM here:
+- [Link to FSM README](fsm/README.md)
+Follow the name setup for the arduino here:
+- [Link to Arduino README](Arduino/README.md)
+
+
+
+Build Everything Together After Dependencies are Installed (Everything is good as long as the build doesn't fail):
+```
+cd ~/ros2_ws
+colcon build --symlink-install
+```
+
+Source before launching
+```
+. install/setup.bash
+```
+
+
+Different Launch Options for TRAILBOT
+=============
+There are 3 different launch modes you can choose to use:
```

Comment:
at least, we also want a mapping only,  and navigation is mapping and navigation.  And we want a bag option for mapping and a sim option for base and navigation